atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/InfoEvent.java

```diff
@@ -40,7 +40,7 @@
 
     public abstract long time();
 
-    public abstract Optional<String> value();
+    public abstract Optional<Object> value();
```

Comment:
I'm not sure about this change. I think the reason this was put in was because the `:info` event for the crashing nemesis wasn't castable as a string. Could you confirm this is the case? Unsure if having `Object` here is going to cause problems elsewhere, although if the tests pass then it should be OK.